docker/rbac.js

```diff
@@ -0,0 +1,223 @@
+import jsyaml from '/js-yaml.js';
+
+var fs = require('fs');
+
+// TODO: ACL location should be parameterized
+var ACL = jsyaml.safeLoad(fs.readFileSync('ACL.yaml'));
+
+var regex = /^\/.*\/$/;
+
+export default {
+  check: check,
+  intercept: intercept,
+  isCanInvokeRequest: isCanInvokeRequest,
+};
+
+var rbacMBean = 'hawtio:area=jmx,type=security';
+
+function isCanInvokeRequest(request) {
+  return request.type === 'exec' && request.mbean === rbacMBean && request.operation === 'canInvoke(java.lang.String)';
```

Comment:
As far as I remember, the client-side Hawtio RBAC plugin uses the following three `canInvoke` methods. Is it ok here to check just one of them?
```js
  const MBEAN_ONLY = 'canInvoke(java.lang.String)';
  const OVERLOADED_METHOD = 'canInvoke(java.lang.String,java.lang.String)';
  const EXACT_METHOD = 'canInvoke(java.lang.String,java.lang.String,[Ljava.lang.String;)';
```

Reply:
Probably this `rbac.js` impl corresponds to `io.hawt.system.RBACMBeanInvoker` from the Hawtio Java backend which only uses `canInvoke(java.lang.String)`?

As for the design decision of `RBACMBeanInvoker`, I remember it comes from the restriction of Jolokia `Restrictor` API, which can only pass the operation name but no arg types/values.

I could always be wrong though :-)

Reply:
From the tests I've been able to run so far, the `canInvoke(java.lang.String,java.lang.String)` and `canInvoke(java.lang.String,java.lang.String,[Ljava.lang.String;)` are not called by `hawtio-integration`. So that explains why I only implemented support for `canInvoke(java.lang.String)`.

So it seems my tests empirically confirms your analysis. Hopefully we are not both empirically and theoretically wrong ;)